### PR TITLE
[Webhook-connectors][correction][1437288]

### PR DESCRIPTION
**Pull request by Mahesh2516**
Removed (A fourth action, `ViewAction`, is still supported but no longer needed; use `OpenUri` instead.) from content. Added updated Json example.

### CODE REVIEW

File: msteams-platform/webhooks-and-connectors/how-to/connectors-using.md [hunk 1]
```diff
@@ -61,7 +61,14 @@ You can also use this JSON to create cards containing rich inputs, such as text
...
             "target": "http://..."
-        }]
+        },
+        {
+      "@type": "OpenUri",
+      "name": "Learn More",
+      "targets": [
+        { "os": "default", "uri": "https://docs.microsoft.com/outlook/actionable-messages" }
+      ]
+    }]
```
Comment (KirtiPereira) [inline msteams-platform/webhooks-and-connectors/how-to/connectors-using.md:71]: Check and correct the alignment of the braces and content with the previous sections braces and content.

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Mahesh2516 - please check and correct the alignment.

File: msteams-platform/webhooks-and-connectors/how-to/connectors-using.md (hunk 1 above)
Comment (Mahesh2516) [reply msteams-platform/webhooks-and-connectors/how-to/connectors-using.md]: Corrected

**Review by Nikitha-MSFT — APPROVED** [review]
Reviewed and approved

**Review by KirtiPereira — APPROVED** [review]
Approved for merging.
Thank you.

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Pull request merged**